tmt-app/src/Components/Stepper.jsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { useLocation } from "react-router-dom";
+import { CustomTheme } from "../Styling/CustomStyling";
+import HomeIcon from "../Images/icons/boba_progress/Home_Current.png";
+import SubjectsIcon from "../Images/icons/boba_progress/Subjects_Current.png";
+import SoftSkillsIcon from "../Images/icons/boba_progress/SoftSkills_Current.png";
+import OrderUpIcon from "../Images/icons/boba_progress/OrderUp_Current.png";
+import {
+  Stepper,
+  Step,
+  StepLabel,
+  StepConnector,
+  stepConnectorClasses,
+  styled,
+} from "@mui/material";
+
+const ColorlibConnector = styled(StepConnector)(({ theme }) => ({
+  [`&.${stepConnectorClasses.active}`]: {
+    [`& .${stepConnectorClasses.line}`]: {
+      backgroundColor: "#1B1D2E",
```

Comment:
Are these colors available in variables from our theme? we have colors below, but would be great to pull them in here too

Reply:
Added in the "Text" color, which wasn't previously in the theme in the repo but was found in Figma.